controller/pom.xml

```diff
@@ -58,5 +58,9 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>rocketmq-srvutil</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+        </dependency>
```

Comment:
Any reasons to add this dependency?

Reply:
> Any reasons to add this dependency?

We need to use log as mertric exporter. 
<img width="1302" alt="image" src="https://github.com/apache/rocketmq/assets/87409330/9c927052-40cb-4eed-824e-5133279279e7">